wiki/Game_modifier/Summary/tr.md

```diff
@@ -0,0 +1,75 @@
+# Oyun modifikasyonlarının özeti
+
+*Ana Sayfa: [Oyun modifikasyonları](/wiki/Game_modifier)*
+
+*Not: osu!mania'da kullanılan modifikasyonlar hiçbir zaman skor çarpanını arttırmayacaktır.*
+
+| Mod | İsim (Kısaltma) | Çarpan | İpucu metni |
+| :-- | :-- | :-- | :-- |
+| ![Easy](/wiki/shared/mods/EZ.png) | Easy (EZ) | • ![][o!s] ![][o!t] ![][o!c] ![][o!m]: 0.50x | • ![][o!s]: `Genel zorluğu düşürür - daha geniş circlelar, HP azalması daha töleranslı ve daha az dikkat gerektirir` • ![][o!t]: `Genel zorluğu düşürür - notalar daha yavaş hareket eder ve daha az dikkat gerektirir.` • ![][o!c]: `Genel zorluğu düşürür - daha geniş circlelar, HP azalması daha töleranslı ve daha az dikkat gerektirir.` • ![][o!m]: `Genel zorluğu düşürür - HP azalması daha töleranslı ve daha az dikkat gerektirir.` | | ![No Fail](/wiki/shared/mods/NF.png) | No Fail (NF) | • ![][o!s] ![][o!t] ![][o!c] ![][o!m]: 0.50x  | • ![][o!s] ![][o!t] ![][o!c] ![][o!m]: `Ne olursa olsun başarısız olamazsın.` |
```

Comment:
```suggestion
| ![Easy](/wiki/shared/mods/EZ.png) | Easy (EZ) | • ![][o!s] ![][o!t] ![][o!c] ![][o!m]: 0.50x | • ![][o!s]: `Genel zorluğu düşürür - daha geniş circlelar, HP azalması daha töleranslı ve daha az dikkat gerektirir` • ![][o!t]: `Genel zorluğu düşürür - notalar daha yavaş hareket eder ve daha az dikkat gerektirir.` • ![][o!c]: `Genel zorluğu düşürür - daha geniş circlelar, HP azalması daha töleranslı ve daha az dikkat gerektirir.` • ![][o!m]: `Genel zorluğu düşürür - HP azalması daha töleranslı ve daha az dikkat gerektirir.` | 
| ![No Fail](/wiki/shared/mods/NF.png) | No Fail (NF) | • ![][o!s] ![][o!t] ![][o!c] ![][o!m]: 0.50x  | • ![][o!s] ![][o!t] ![][o!c] ![][o!m]: `Başarısız olamazsınız. Ne olursa olsun.` |
```
umm, bu tablolar neden aynı satırda?

Reply:
bir hatayla yanlış file da boğuştuğum için